### ISSUE TITLE
The model and loaded state dict do not match exactly

### ISSUE THREAD

**Issue by LUO77123** [SwinTransformer/Swin-Transformer-Object-Detection#132]
**Notice**
mmcv 1.3.17
mmdet 2.11

The model and loaded state dict do not match exactly

**Describe the issue**

oad checkpoint from local path: mask_rcnn_swin_tiny_patch4_window7.pth
The model and loaded state dict do not match exactly

size mismatch for roi_head.bbox_head.fc_cls.weight: copying a param with shape torch.Size([81, 1024]) from checkpoint, the shape in current model is torch.Size([3, 1024]).
size mismatch for roi_head.bbox_head.fc_cls.bias: copying a param with shape torch.Size([81]) from checkpoint, the shape in current model is torch.Size([3]).
size mismatch for roi_head.bbox_head.fc_reg.weight: copying a param with shape torch.Size([320, 1024]) from checkpoint, the shape in current model is torch.Size([8, 1024]).
size mismatch for roi_head.bbox_head.fc_reg.bias: copying a param with shape torch.Size([320]) from checkpoint, the shape in current model is torch.Size([8]).
size mismatch for roi_head.mask_head.conv_logits.weight: copying a param with shape torch.Size([80, 256, 1, 1]) from checkpoint, the shape in current model is torch.Size([2, 256, 1, 1]).
size mismatch for roi_head.mask_head.conv_logits.bias: copying a param with shape torch.Size([80]) from checkpoint, the shape in current model is torch.Size([2]).
/home/zhuyu/anaconda3/envs/py39torch/lib/python3.9/site-packages/mmdet-2.11.0-py3.9.egg/mmdet/datasets/utils.py:64: UserWarning: "ImageToTensor" pipeline is replaced by "DefaultFormatBundle" for batch inference. It is recommended to manually replace it in the test data pipeline in your config file.
  warnings.warn(
/home/zhuyu/anaconda3/envs/py39torch/lib/python3.9/site-packages/torch/nn/functional.py:718: UserWarning: Named tensors and all their associated APIs are an experimental feature and subject to change. Please do not use them for anything important until they are released as stable. (Triggered internally at  /pytorch/c10/core/TensorImpl.h:1156.)
  return torch.max_pool2d(input, kernel_size, stride, padding, dilation, ceil_mode)

**Comment by nestiank** [comment]
This warning occurred because the number of classes in your config file is 3, not 81 like COCO dataset.

For those layers listed in the warning message, pretrained weights loading process will be skipped.

For the layers not listed in the warning message, pretrained weights will be loaded.
